include/deal.II/particles/generators.h

```diff
@@ -207,6 +210,8 @@ namespace Particles
                        ParticleHandler<dim, spacedim> &particle_handler,
                        const Mapping<dim, spacedim> &  mapping =
                          StaticMappingQ1<dim, spacedim>::mapping,
+                       const std::vector<std::vector<double>> &properties =
```

Comment:
Could you switch the order of `properties` and `components`. I think we need that for backwards  compatibility.